src/python/deepgnn/graph_engine/snark/convert.py

```diff
@@ -37,7 +35,7 @@
 
 
 class _NoOpWriter:
-    def add(self, _: typing.Any):
+    def add(self, *_: typing.Any):
```

Comment:
why do we need this change?

Reply:
NoOpWriter is a mock of Node/EdgeAliasWriter, I changed the add functions of those classes to use the individual values required(src, dst, weight) instead of a dictionary.